cloud/aws/services/ec2/service.go

```diff
@@ -15,13 +15,15 @@ package ec2
 
 import (
 	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
+	"github.com/aws/aws-sdk-go/service/elb/elbiface"
```

Comment:
Should this maybe directly start with ELBv2 aka NLBs or ALBs?

Reply:
We discussed this as we were scoping out the MVP and decided to go with an ELB Classic configured for tcp-passthrough. The main reason for this is to avoid hitting hairpining-related issues when an API server attempts to talk to itself through the NLB.

If we can sufficiently work through those issues, then we can move towards an NLB-based approach. ALB is more difficult because we need to allow client-cert based authentication.

Reply:
Thanks for the chiming in @detiber.
Yes, this was captured in #52 
> The load balancer should be a classic ELB configured for tcp-passthrough.